robot/Cumulus/resources/locators_51.py

```diff
@@ -340,7 +340,8 @@
         'batch_error':'//div[@role="alert" and contains(@class,"slds-theme--error")]//div[text()="{}"]/following-sibling::span[text()="{}"]',
         'close_error':'//div[@role="alert" and contains(@class,"slds-theme--error")]//child::button',
         'perms_error':'//div[@class="slds-illustration slds-illustration_large"]',
-        'template_required_checkbox':'//lightning-input[@data-qa-locator="checkbox Required {}"]'
+        'template_required_checkbox':'//lightning-input[@data-qa-locator="checkbox Required {}"]',
+        'datepicker':'//*[contains(@data-qa-locator,"{}")]//child::lightning-datepicker'
```

Comment:
please remove this if its not needed anymore

Reply:
Removed